lib/get_navigation/src/extension_navigation.dart

```diff
@@ -220,7 +220,7 @@ extension ExtensionDialog on GetInterface {
     Color barrierColor,
     bool useSafeArea = true,
     bool useRootNavigator = true,
-    RouteSettings routeSettings,
+    Object arguments,
```

Comment:
Unfortunately this cannot be changed, first because it would be a break update, and second that it is possible to define Dialog names via routeSettings, something that is not very common, but it is a feature.